src/platform/packages/private/kbn-language-documentation/src/sections/generated/aggregation_functions.tsx

```diff
@@ -168,18 +156,15 @@ export const functions = {
           markdownContent={i18n.translate(
             'languageDocumentation.documentationESQL.median.markdown',
             {
-              defaultMessage: `<!--
-  This is generated by ESQL’s AbstractFunctionTestCase. Do no edit it. See ../README.md for how to regenerate it.
-  -->
-
+              defaultMessage: `
   ### MEDIAN
-  The value that is greater than half of all values and less than half of all values, also known as the 50% \`PERCENTILE\`.
+  The value that is greater than half of all values and less than half of all values, also known as the 50% [\`PERCENTILE\`](https://www.elastic.co/docs/reference/elasticsearch/query-languages/esql/functions-operators/aggregation-functions#esql-percentile).
```

Comment:
```suggestion
  The value that is greater than half of all values and less than half of all values, also known as the 50% [\`PERCENTILE\`](https://www.elastic.co/docs/reference/query-languages/esql/functions-operators/aggregation-functions#esql-percentile).
```

Reply:
applies to all these :)

Reply:
Thanx Liam, waiting for ES to update the links and we will update